lib/opensusebasetest.pm

```diff
@@ -82,6 +82,7 @@ this method.
 
 sub post_run_hook {
     my ($self) = @_;
+    $self->record_avc_selinux_alerts();
```

Comment:
Do NOT, under any circumstances, put any code in `opensusebasetest::post_run_hook()`. Whatever you put in here will be executed after every single test module in every single job. And in some cases, it's a point where the SUT is rebooting!
https://openqa.suse.de/tests/18203266#step/update_kernel/81